pkg/kubectl/resource/visitor.go

```diff
@@ -396,19 +396,21 @@ func ExpandPathsToFileVisitors(mapper *Mapper, paths string, recursive bool, ext
 			if path != paths && !recursive {
 				return filepath.SkipDir
 			}
-			return nil
```

Comment:
@deads2k @AdoHe you are both correct. This section does not need to be edited - it was late last night when I thought that change was necessary. apologies for the confusion